doc/ref/methsel.xml

```diff
@@ -68,6 +68,71 @@ Depending on the type of the object,
 there may be another method to store the value in a suitable way,
 and then set the attribute tester for the object to <K>true</K>.
 
+<#Include Label="IsOperation">
+<#Include Label="TypeOfOperation">
+<#Include Label="ShowDeclarationsOfOperation">
+<#Include Label="NewOperation">
+<#Include Label="DeclareOperation">
+
+</Section>
+
+
+<!-- %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% -->
+<Section Label="Constructors">
+<Heading>Constructors</Heading>
+
+Constructors are a special type of operation used to make new
+objects. The key difference is that the first argument in a call to a
+constructor is a filter <Ref Sect="Filters"/> rather than an
+object. This signifies filters in which the constructed object must
+lie and method selection is based on the value of this filter, rather
+than its type.
```

Comment:
"rather than its type" is very hard to understand since "type" is unclear. From the first line of the paragraph I gather that "Constructor" is a type but this is probably not meant. I suggest replacing "than its type" by "than the types of its inputs" or something along those lines to make it clearer. Then, I would probably change the "type" from the first line into "kind".

Reply:
No, "Constructor" is not a type. You are right, this is hard to understand. What is really meant that, while method selection for operations depends on the types of the arguments, this is not the case for constructors... Anyway, I'll try to write a better text.

Reply:
I rewrote that text, please have another look.

Reply:
Yes, it is much clearer now.